src/git.js

```diff
@@ -131,19 +131,41 @@ export default class Git {
 		const prefix = BRANCH_PREFIX.replace('SOURCE_REPO_NAME', GITHUB_REPOSITORY.split('/')[1])
 
 		let newBranch = path.join(prefix, this.repo.branch).replace(/\\/g, '/').replace(/\/\./g, '/')
+		this.prBranch = newBranch
 
 		if (OVERWRITE_EXISTING_PR === false) {
 			newBranch += `-${ Math.round((new Date()).getTime() / 1000) }`
+			this.prBranch = newBranch
+
+			core.debug(`Creating PR Branch ${ newBranch }`)
+
+			await execCmd(
+				`git switch -b "${ newBranch }"`,
+				this.workingDir
+			)
+
+			return
 		}
 
-		core.debug(`Creating PR Branch ${ newBranch }`)
+		core.debug(`Switch/Create PR Branch ${ newBranch }`)
 
 		await execCmd(
-			`git checkout -b "${ newBranch }"`,
+			`git remote set-branches origin '*'`,
 			this.workingDir
 		)
 
-		this.prBranch = newBranch
+		await execCmd(
+			`git fetch -v --depth=1`,
+			this.workingDir
+		)
```

Comment:
The code currently only checks out the `default` branch. We need to fetch the other branches so we can switch to as required.